feature-group-definitions/flexbox.yml

```diff
@@ -1,3 +1,59 @@
 spec: https://drafts.csswg.org/css-flexbox-1/
 caniuse: flexbox
 usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/1692
+status:
+  is_baseline: true
+  since: "2020-10-20"
+  support:
+    chrome: "57"
```

Comment:
This and the below are quite a lot later than https://caniuse.com/flexbox. What are the newest part of the feature that cause the whole feature to look new here?

Reply:
The key thing affecting the date is Firefox 81's full support for `flex-direction`. Before that it has partial support. That feature alone determines the baseline date (without it, it would be determined by Edge 80's release date).

For the individual browsers' differences with caniuse, let's look at each in turn. In general, I ignored prefixed-only and partial implementations, but this has a smaller effect than you might guess.

For Chrome, most of the features lined up with caniuse's 29+. The exceptions were:

- Chrome 57: `css.properties.align-content.flex_context.stretch`, `css.properties.align-self.flex_context.stretch`, `css.properties.justify-content.flex_context.stretch`
- Chrome 36: `css.properties.align-self.flex_context`

For Edge, both the `stretch` and `baseline` values push it above caniuse's Edge 12 to Edge 79. I also treated `≤79` as Edge 79, though I don't think it would make a difference to the sum of the features.

For Firefox, most of the features were roughly in line with caniuse, though caniuse says that before Firefox 28 is partial, while BCD uses notes with no partial flag. Beyond Firefox 28, the divergences from caniuse are:

- Firefox 52: `stretch` values, again
- Firefox 81: `css.properties.flex-direction` stops being a partial implementation. This one is messy, with a partial and notes.

For Safari, caniuse shows a green indicator for prefixed implementations starting at Safari 7. Waiting for the unprefixed versions pushes most of the features to Safari 9; I think that's the only thing we can do before sorting out ways to annotate results.

Taking this all together:

- If Firefox's data for `flex-direction` is wrong or misleading, then corrections might lead to the overall baseline date back to Edge 80's release, about 8 months earlier.
- If that were true *and* EdgeHTML had more complete flexbox support, then it's possible the date could shift backwards even further, to before 2020.
- I'm not a subject matter expert, but it's possible that `stretch` is a separate group. If it were moved to another group (and nothing else changed), then `flexbox` would show an earlier version for Chrome (36), but wouldn't otherwise alter the results.

Reply:
Alright, so clearly there's a lot of nuance here, a cluster of granular CSS properties and values, many of which come with a complex history of partial support, notes, etc.

I'm happy with the explanation of the versions you've given, and the only issue I can foresee is if any consumer uses the version numbers and displays them to users. There are multiple ways we can address that, but for now I think we should just proceed with the manually determined versions you have here.